package.json

```diff
@@ -12,7 +12,9 @@
     "packages/falso"
   ],
   "dependencies": {
-    "tslib": "2.0.0"
+    "seedrandom": "^3.0.5",
```

Comment:
`seedrandom` already was added in another PR that is merged already, please sync branch with main

Reply:
@iamandrewluca removing this from the global `package.json` breakes one test case (also in the pre-commit).
![image](https://user-images.githubusercontent.com/21980644/149038197-a67762d5-b2c0-4724-b19c-e45a874354ef.png)